File: ExtremeCore/Classes and Modules/stringSearcher.vb
Public Class StringSearcher
    Public Shared Function CountChar(src As String, chr As Char) As Integer
        Dim num As Integer = 0

        For Each fChr As Char In src
            If fChr = chr Then
                num += 1
            End If
        Next

        Return num
    End Function
End Class

File: ExtremeStudio.Core/Classes and Modules/stringSearcher.cs
using System.Linq;

namespace ExtremeStudio.Core.Modules
{
	public class StringSearcher
	{
		public static int CountChar(string src, char chr)
		{
			return System.Convert.ToInt32(src.Count(fChr => fChr == chr));
		}
	}
	
}
